src/views/self/view_teleport.cpp

```diff
@@ -82,6 +84,7 @@ namespace big
 		});
 		ImGui::EndGroup();
 
+
 		ImGui::SeparatorText("VEHICLES"_T.data());
 		ImGui::Spacing();
 
```

Comment:
Revert these changes, the code doesn't compile because there's no "core/data/teleport_presets.hpp" file.